src/engine/TextIndexScanForEntity.cpp

```diff
@@ -126,8 +126,11 @@ vector<ColumnIndex> TextIndexScanForEntity::resultSortedOn() const {
 
 // _____________________________________________________________________________
 string TextIndexScanForEntity::getDescriptor() const {
+  std::ostringstream oss;
+  oss << config_;
   return absl::StrCat("TextIndexScanForEntity on ",
-                      config_.varToBindText_.name());
+                      config_.varToBindText_.name(),
+                      " With config: ", oss.str());
```

Comment:
The descriptors should be short.
In particular, they should not contain information from the `VariableToColumnMap`,
so e.g. the bindMatch and bindScore variables can be omitted.
If you want to add more verbose information,
you should add it to the runtime Information via `addDetail`
(That would be a good idea for complex configurations). Those are shown in the runtime tree when hovering over the node.

(same goes for the TextIndexScanForWord).